src/Core/System/Consent/Event/ConsentAcceptedEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Consent\Event;
+
+use Shopware\Core\Framework\App\AppEntity;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Webhook\AclPrivilegeCollection;
+use Shopware\Core\Framework\Webhook\Hookable;
+use Shopware\Core\System\Consent\ConsentScope;
+
+#[Package('data-services')]
+readonly class ConsentAcceptedEvent implements Hookable
+{
+    public const EVENT_NAME = 'consent.accepted';
+
+    public function __construct(
+        public string $consentName,
+        public ConsentScope $consentScope,
+        public string $identifier
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return self::EVENT_NAME;
+    }
+
+    /**
+     * @return array{consent: string, scope: string, identifier: string|null}
+     */
+    public function getWebhookPayload(?AppEntity $app = null): array
+    {
+        return [
+            'consent' => $this->consentName,
+            'scope' => $this->consentScope->value,
+            'identifier' => $this->identifier,
+        ];
+    }
+
+    public function isAllowed(string $appId, AclPrivilegeCollection $permissions): bool
```

Comment:
should we add custom ACL permissions here as well, to make clear that an app might receive consent state changes?